internal/store/pod.go

```diff
@@ -1433,6 +1461,32 @@ func createPodStatusReadyFamilyGenerator() generator.FamilyGenerator {
 	)
 }
 
+func createPodStatusReadyTimeFamilyGenerator() generator.FamilyGenerator {
+    return *generator.NewFamilyGenerator(
+        "kube_pod_status_ready_time",
```

Comment:
But we already have kube_pod_status_ready, which is a timeseries entry in Prometheus. What would be your need for timestamp here curious?

Reply:
According to [docs](https://github.com/kubernetes/kube-state-metrics/blob/master/docs/pod-metrics.md) kube_pod_status_ready is like a boolean, 0 or 1. `kube_pod_status_ready_time` would inform you when was the last time pod entered `Ready` state.